docs/examples/go-mod.ort.yml

```diff
@@ -1,5 +1,5 @@
 excludes:
   scopes:
     - pattern: "all"
-      reason: "BUILD_TOOL_OF"
+      reason: "DEV_DEPENDENCY_OF"
```

Comment:
Shouldn't this be `BUILD_DEPENDENCY_OF`?

Reply:
comment from @tsteenbe  : dev dependency of is correct as Go Mod to my knowledge mixes test and build deps so safest option is dev dependency of.